### ISSUE TITLE
Intermittently when updating NVDA, nvda_slave.exe sometimes does not exit

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3890]
**Reported by twynn92 on 2014-02-17 05:23**
Every so often when updating NVDA, I intermittently receive an error,due to nvda_slave.exe not exiting. Doing a manual closing of nvda_slave.exe from the processes tab of the windows task manager in Windows XP SP3, then pressing the "Retry" button succeeds in the update.

I've no idea how NVDA closes nvda_slave.exe before exiting, but this error only occurs every so often. Here's a fragment of the log:
ERROR - RPC process 4316 (nvda_slave.exe) (00:03:58):
**main**.main:
Task failed, try again
Traceback (most recent call last):
  File "nvda_slave.pyw", line 29, in main
  File "installer.pyc", line 348, in install
  File "installer.pyc", line 311, in tryRemoveFile
RetriableFailure: File C:\Program Files\NVDA\nvda_slave.exe could not be removed
ERROR - gui.installerGui.doInstall (00:03:59):
Failed to execute installer
Traceback (most recent call last):
  File "gui\installerGui.pyc", line 32, in doInstall
RetriableFailure

Obviously this is a minor bug, but I thought it worth reporting anyway as I couldn't find an existing ticket for this issue.

**Comment by bhavyashah** [comment]
@jcsteh You found the reported issue particularly odd initially, and the original author of this ticket, although now out-of-reach, did respond in https://github.com/nvaccess/nvda/issues/3890#issuecomment-155317516. I am unsure of how this ticket should be triaged.

**Comment by Brian1Gaff** [comment]
I have a machine that occasionally leaves an open window when it runs, which 
is not actually actionable, but attempting to close it asks if you wish to 
exit nvda. It does happen more after an update but can happen at start up. 
Windows 7 64 bit. I've no idea what exactly is not exiting cleanly, but 
normally a reboot of nvda will clear it  in the end.
 Its not the only piece of software with strange start up quirks though, so 
I suspect its a windows issue perhaps when a processor core is being delayed 
for whatever reason.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.

**Comment by jcsteh** [comment]
Closing as works for me, as we haven't had any report of this in several years and I can't reproduce it.
